Templates/SasToken-CSharp/run.csx

```diff
@@ -0,0 +1,97 @@
+// An HTTP trigger Azure Function that returns a SAS token for Azure Storage for the specified container. 
+// You can also optionally specify a particular blob name and access permissions. 
+// To learn more, see https://github.com/Azure/azure-webjobs-sdk-templates/blob/master/Templates/SasToken-CSharp/readme.md
+
+#r "Microsoft.WindowsAzure.Storage"
+
+using System.Net;
+using System.Configuration;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+public static async Task<HttpResponseMessage> Run(HttpRequestMessage req, TraceWriter log)
+{
+    dynamic data = await req.Content.ReadAsAsync<object>();
+
+    if (data.container == null) {
+        return req.CreateResponse(HttpStatusCode.BadRequest, new {
+            error = "Specify value for 'container'"
+        });
+    }
+
+    var permissions = SharedAccessBlobPermissions.Read; // default to read permissions
+    Enum.TryParse(data.permissions.ToString(), out permissions);
+
+    var storageAccount = CloudStorageAccount.Parse(ConfigurationManager.AppSettings["AzureWebJobsStorage"]);
```

Comment:
You could simplify this example by adding a `CloudStorageAccount account` parameter to your function signature. The WebJobs SDK will then pass it in for you automatically. Of course that only works if you're targeting the "AzureWebJobsStorage" default account.